testsuite/integration/clustering/pom.xml

```diff
@@ -986,8 +986,7 @@
                                                 <layer>jpa</layer>
                                             </excluded-layers>
                                             <layers>
-                                                <layer>cloud-server</layer>
-                                                <layer>web-clustering</layer>
+                                                <layer>jaxrs-server</layer>
                                                 <layer>h2-default-datasource</layer>
                                                 <layer>jpa-distributed</layer>
```

Comment:
Notice I've also picked up a slimmer layer configuration for the jpa clustering tests. These tests are only using the hibernate container cache exposed by jaxrs resources, so we need neither `web-clustering` nor `cloud-server` to run it.

Reply:
OK.